### PR TITLE
Adds tests for ConstValueDist constructed using a NamedTuple

### PR DESCRIPTION
**Pull request by sthayashi**

### CODE REVIEW

File: test/test_const_value_dist.jl [hunk 1]
```diff
@@ -77,4 +77,23 @@ using Random
     @test @inferred(mode(univariate_cvd)) == 42
 
     @test @inferred(eltype(univariate_cvd)) == Int64
+    
+    μ1, μ2 = rand(2)
+    cvd_from_named_tuple = @inferred(ConstValueDist((a=μ1, b=μ2)))
```
Comment (sthayashi) [inline test/test_const_value_dist.jl:82]: The constructor of a `ConstValueDist` when using a named tuple may not be strict enough. For example, something like
```
const_value_dist_from_scalars = ConstValueDist((a=0.0, b=0.0))
const_value_dist_from_normals = ConstValueDist((a=Normal(), b=Normal()))

# Works as expected
pdf(const_value_dist_from_scalars, (a=0.0, b=0.0)) == Inf

# Not sure about these two pdf evaluations
pdf(const_value_dist_from_normals, (a=0, b=0)) == 0.0
pdf(const_value_dist_from_normals, (a=Normal(), b=Normal())) == Inf
```
Not sure what it means to evaluate a distribution at a distribution. Is this intended to work as shown above?

**Comment by sthayashi** [comment]
A test for https://github.com/oschulz/ValueShapes.jl/blob/32dba1b02f61cf794c3a126461aa4c4f0e01f05a/src/const_value_dist.jl#L46 isn't added, because it sounds like it may be changed in the future. Easy enough to add a simple tests. Otherwise this PR is ready.

File: test/test_const_value_dist.jl (hunk 1 above)
Comment (oschulz) [reply test/test_const_value_dist.jl:82]: The value wrapped in `ConstValueDist` should (typically) *not* include distributions. A `ConstValueDist` is a dirac-style-distribution (like the new `Distributions.Dirac`, we'll have to see if we can switch to using that in the future, possibly with some additions).

**Comment by oschulz** [comment]
Good to merge?

**Comment by sthayashi** [comment]
I'll say yes, and fix the `ConstValueDist` constructor in a different PR.

**Comment by oschulz** [comment]
Ok, I'll merge then.

**Pull request merged**